src/edu/csus/ecs/pc2/services/eventFeed/WebServer.java

```diff
@@ -511,4 +486,86 @@ public boolean isServerRunning() {
         return serverRunning;
     }
 
+
+    /**
+     * Account Listener for updating realm accounts.
+     *
+     * @author John Buck
+     */
+
+    protected class AccountListener implements IAccountListener {
+
+        @Override
+        public void accountAdded(AccountEvent accountEvent) {
+            addBasicAuthInfo(accountEvent.getAccount());
+        }
+
+        @Override
+        public void accountModified(AccountEvent accountEvent) {
+            addBasicAuthInfo(accountEvent.getAccount());
+        }
+
+        @Override
+        public void accountsAdded(AccountEvent accountEvent) {
+            Account[] accounts = accountEvent.getAccounts();
+            for (Account account : accounts) {
+                addBasicAuthInfo(account);
+            }
+        }
+
+        @Override
+        public void accountsModified(AccountEvent accountEvent) {
+            Account[] accounts = accountEvent.getAccounts();
+            Arrays.sort(accounts, new AccountComparator());
```

Comment:
Is there a reason to sort the accounts array before adding each account?   Seems wasteful if not...